templates/tour.html

```diff
@@ -292,7 +360,7 @@ <h3 class="p-heading--four">Configure any storage layout no matter how complex</
               <img class="p-takeunder__image p-takeunder--right__image" src="{{ ASSET_SERVER_URL }}4508bba8-maas-cli.svg" width="150" alt="MAAS command line interface icon" />
             </div>
             <div class="col-8">
-              <h3><a class="p-heading--four p-link--inverted" href="https://docs.ubuntu.com/maas/2.2/en/manage-cli" onClick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Feature', 'eventAction' : 'Click', 'eventLabel' : 'REST API, CLI' });" aria-label="External link to manage command line interface page on MAAS documentation">See MAAS <abbr title="Command line interface">CLI</abbr> on how to get started with the&nbsp;<abbr title="Command line interface">CLI</abbr>&nbsp;&rsaquo;</a></h3>
+              <h3><a class="p-heading--four p-link--inverted" href="https://docs.ubuntu.com/maas/2.2/en/manage-cli" onClick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Feature', 'eventAction' : 'Click', 'eventLabel' : 'REST API, CLI' });" aria-label="External link to manage command line interface page on MAAS documentation">See MAAS CLI on how to get started with the&nbsp;CLI&nbsp;&rsaquo;</a></h3>
```

Comment:
Do you need the &nbsp;'s around CLI?

Reply:
The one before CLI would be so you don't get an orphan when it wraps, it'll always stick with 'the' same thing for the one after, it's so you're not left with an orphaned '&rsaquo;', however, I'm not sure we need the latter as the former possible negates its existence